lib/private/Files/Template/TemplateManager.php

```diff
@@ -275,6 +275,11 @@ public function initializeTemplateDirectory(string $path = null, string $userId
 		$isDefaultTemplates = $skeletonTemplatePath === $defaultTemplateDirectory;
 		$userLang = $this->l10nFactory->getUserLanguage($this->userManager->get($this->userId));
 
+		if (empty($skeletonTemplatePath)) {
```

Comment:
```suggestion
		if ($skeletonTemplatePath === '') {
```

Since you use `getSystemValueString` it only returns string, then you can explicitly check for the empty string